### DISCUSSION TITLE
What's the latest release? (Possible update channel issue?)

### DISCUSSION THREAD

**Discussion by Lupum1001** [NightscoutFoundation/xDrip#2145] [General]
I'm starting to wonder if my installation has stopped receiving updates. It's currently still running the October 4th build from last year.
I noticed that the latest release of xDrip is marked as Jan 31st. I assume that's at least a Beta build, but I can't get xDrip to detect it.
<img width="300" src="https://user-images.githubusercontent.com/52087143/170383841-67dc4918-0af9-4309-af04-5b167c933f1f.png">
I've tried the Stable, Beta, _and_ Alpha channels, but none of them show an available update. The Nightly channel does, but that's it.

---

**UPDATE:** I reverse-engineered the url, and determined that the Stable/Beta/Alpha urls _DO_ point to the January release, so I'm not entirely sure why xDrip isn't seeing it.
https://xdrip-plus-updates.appspot.com/update-check/beta
```
2201310800
https://github.com/NightscoutFoundation/xDrip/releases/download/2022.01.31/xDrip-plus-20220131-33be05a.apk?
12779931

c28cc7f8786c7147876caf0cd869af33e0a8e09a4f4bfc8ffe7e2c1456bd36db
```
https://xdrip-plus-updates.appspot.com/update-check/nightly
```
2122051908
https://github.com/NightscoutFoundation/xDrip/releases/download/2022.05.19/xDrip-plus-20220519-e557cba.apk?
13050888

44b24ed125a1425b9497af9136e31355e1ec9ec8dc477110ec8c2572685bce19
```

**Comment by Navid200** [comment]
I can confirm.  I tried versions going all the way back to September 2020 with the same result.

Would you please open an issue?

**Comment by Navid200** [comment]
Currently, alpha, beta and stable all are January 31, 2022.

**Comment by Lupum1001** [comment]
created a new issue here:
https://github.com/NightscoutFoundation/xDrip/issues/2146

**Comment by Navid200** [comment]
This will be fixed tomorrow (Monday, June 13, 2022).
